### PR TITLE
docs: add English version of Java SDK quickstart documentation

### LINKED ISSUE #1322: docs: translate the Java SDK quickstart from Chinese to English
1. Translate the Java SDK from Chinese to English: https://docs.openmldb.ai/quickstart/java_sdk
The source of this doc is here: https://github.com/4paradigm/openmldb-docs-zh/blob/dev/quickstart/java_sdk.md
2. Save the translated English version in this repo as docs/en/java_sdk.md

### PR DESCRIPTION
**Pull request by cswjd**
feat: add English version of Java SDK quickstart documentation

### CODE REVIEW

**Comment by cswjd** [comment]
close #1322

File: docs/en/java_sdk.md [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+```java
+SdkOption option = new SdkOption();
+option.setZkCluster("127.0.0.1:2181");
+option.setZkPath("/openmldb");
+option.setSessionTimeout(10000);
+option.setRequestTimeout(60000);
+
+```
+
+Next, create an Executor using SdkOption. SqlClusterExecutor is multi-threaded safe to execute SQL operations. In the actual environment, just create one `SqlClusterExecutor`:
```
Comment (lumianph) [inline docs/en/java_sdk.md]: multi-threaded safe => thread-safe

File: docs/en/java_sdk.md [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+      result.close();
+    }
+  } catch (SQLException throwables) {
+    throwables.printStackTrace();
+  }
+
+}
+```
+
+### 2.6 Execute SQL on-demand queries
```
Comment (lumianph) [inline docs/en/java_sdk.md]: SQL Queries in the request mode

File: docs/en/java_sdk.md [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+
+### 2.8 Delete database
+
+Use the `SqlClusterExecutor::dropDB(db)` interface to drop the specified database:
+
+```java
+sqlExecutor.dropDB(db);
+```
+
+## 3. Complete Java SDK usage example
```
Comment (lumianph) [inline docs/en/java_sdk.md]: A complete Java SDK usage example

File: docs/en/java_sdk.md (hunk 2 above)
Comment (cswjd) [reply docs/en/java_sdk.md]: ok, thank you.

File: docs/en/java_sdk.md (hunk 3 above)
Comment (cswjd) [reply docs/en/java_sdk.md]: ok, thank you.

File: docs/en/java_sdk.md (hunk 1 above)
Comment (cswjd) [reply docs/en/java_sdk.md]: ok, thank you.

**Review by lumianph — APPROVED** [review]

**Pull request merged**